src/models/ArenaRound.ts

```diff
@@ -122,7 +123,7 @@ export class ArenaRound
   _actions?: ArenaRoundAction[];
 
   static associations: {
-    _game: Association<ArenaRound, Game>;
+    _game: Association<ArenaRound, Game>; // TBD association with ArenaGame?
```

Comment:
this association is a bug that I found. 
changing it to ArenaGame involves some non-trivial changes that I preferred leaving out of this PR.
However, if we'd want to make an ArenaGame scalable and allow more than one active game, we need to fix this otherwise the findActiveRound() could start returning rounds that don't belong to the corresponding ArenaGame